pegin/pegin.go

```diff
@@ -18,19 +24,12 @@ import (
 	"github.com/btcsuite/btcd/btcec"
 )
 
-const (
-	MainNet NetworkType = iota
-	RegtestNet
-)
-
-type NetworkType int
-
 type AddressInfo struct {
 	ClaimScript      string
 	MainChainAddress string
 }
 
-func PeginAddress(
+func MainChainAddress(
```

Comment:
This will eventually break the wasm package module it depends on it. Any reason to change the API name?

Reply:
Package name is already named and its not good practice that method contains pkg name.

Reply:
Should i still use PeginAddress?

Reply:
I see the point, just that now we depend on it. But since we touch again to add the Claim method, its ok